fem/fespace_ext.hpp

```diff
@@ -20,24 +20,22 @@ namespace mfem
 {
 
 // ***************************************************************************
-// * kFiniteElementSpace
-//  **************************************************************************
-class kFiniteElementSpace
+// * FiniteElementSpaceExtension
+// ***************************************************************************
+class FiniteElementSpaceExtension
 {
 private:
    FiniteElementSpace *fes;
    int globalDofs, localDofs;
```

Comment:
Similarly, I think that `globalDofs` should be renamed `localDofs` or `lVectorDofs`, and `localDofs` should be renamed `elementDofs` or `eVectorDofs`.